openassessment/management/tests/test_collect_ora2_data.py

```diff
@@ -69,14 +69,14 @@ def setUp(self):
             "i4x://edX/DemoX/openassessment/hash_value",
             "e31b4beb3d191cd47b07e17735728d53",
             "2014-10-07 20:33:31+00:00",
-            '{""text"": ""This is a response to a question. #dylan""}',
+            u'{""text"": ""This is a response to a question. #dylan""}',
             "Assessment #1 -- scored_at: 2014-10-07 20:37:54 -- type: T -- scorer_id: hash -- feedback: Test",
             "Assessment #1 -- Content: Unclear recommendation (5)",
             "2014-10-07 21:35:47+00:00",
             "10",
             "20",
             "Completed test assessments.",
-            "\xf0\x9d\x93\xa8\xf0\x9d\x93\xb8\xf0\x9d\x93\xbe",
+            "𝓨𝓸𝓾",
```

Comment:
Should all these strings be prefixed with a `u` instead of just the ones you have?

Reply:
thats submission api data. user inputs feedback, and raw_answer could be unicode but changing to other data causes problems.